src/main/kotlin/io/kuzzle/sdk/protocol/Http.kt

```diff
@@ -23,12 +30,132 @@ open class Http : AbstractProtocol {
         isSsl: Boolean = false
     ) {
         if (!isSsl) {
-            this.uri = "http://$host:$port/_query"
+            this.uri = "http://$host:$port"
         } else {
-            this.uri = "https://$host:$port/_query"
+            this.uri = "https://$host:$port"
+        }
+        super.addListener<LoginAttemptEvent>(::onLoginAttempt)
```

Comment:
<LoginAttemptEvent> can be removed

Reply:
I dont see what oculd be removed here, there is a difference between the URIs one has `http` the other has `https`

Reply:
Sorry: the <LoginAttemptEvent>, like in Java, it's implicit.

Reply:
Ah right, I will remove it thanks 😄